app/Http/Controllers/User/BookController.php

```diff
@@ -19,19 +21,26 @@ class BookController extends Controller
 
     protected $media;
 
+    protected $review;
+
     protected $with = [
         'medias',
         'categories',
         'owners',
         'reviews',
     ];
 
-    public function __construct(BookRepository $book, CategoryRepository $category, BookCategoryRepository $bookCategory, MediaRepository $media)
-    {
+    public function __construct(BookRepository $book,
+    CategoryRepository $category,
+    BookCategoryRepository $bookCategory,
+    MediaRepository $media,
+    ReviewBookRepository $review
+    ) {
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.Indent</br>**Severity**: error</br>**File**: app/Http/Controllers/User/BookController.php</br>Multi-line function declaration not indented correctly; expected 8 spaces but found 4